Loop/Managers/Alerts/InAppModalAlertIssuer.swift

```diff
@@ -93,15 +93,18 @@ extension InAppModalAlertIssuer {
             if self.isAlertShowing(identifier: alert.identifier) {
                 return
             }
-            self.playSound(for: alert)
-            let alertController = self.presentAlert(title: content.title,
-                                                    message: content.body,
-                                                    action: content.acknowledgeActionButtonLabel,
-                                                    isCritical: content.isCritical) { [weak self] in
-                self?.clearDeliveredAlert(identifier: alert.identifier)
-                self?.alertManagerResponder?.acknowledgeAlert(identifier: alert.identifier)
+            let alertController = self.constructAlert(title: content.title,
+                                                      message: content.body,
+                                                      action: content.acknowledgeActionButtonLabel,
+                                                      isCritical: content.isCritical) { [weak self] in
+                  self?.clearDeliveredAlert(identifier: alert.identifier)
+                  self?.alertManagerResponder?.acknowledgeAlert(identifier: alert.identifier)
+            }
+            self.alertPresenter?.present(alertController, animated: true) { [weak self] in
+                // the completion is called after the alert is displayed
+                self?.playSound(for: alert)
```

Comment:
I moved `playSound` into the completion such that the sound will not play unless the alert is displayed. This means the alert is displayed first and then the sound plays.

Reply:
Is there much of a delay?  If not, then this is fine by me 👍

Reply:
It is noticeable (<= ~1 sec in my experience), but not significant from my point of view. I think it is only noticeable since I was used to the sound playing before the alert was displayed. 

I prefer it this way, since now when the sound plays, there is something for the user to look at. Previously, if the user was looking at the app when the alert was triggered, the sound would play and the reason why came shortly afterwards.